uber/tests/models/test_attendee.py

```diff
@@ -398,7 +398,7 @@ def test_set_comped_to_complete(self):
     def test_set_group_paid_to_complete(self, monkeypatch):
         monkeypatch.setattr(Group, 'amount_unpaid', 0)
         g = Group()
-        a = Attendee(paid=c.PAID_BY_GROUP, badge_status=c.NEW_STATUS, first_name='Paid', placeholder=False, group=g)
+        a = Attendee(paid=c.PAID_BY_GROUP, badge_status=c.NEW_STATUS, first_name='Paid', placeholder=False, group=g, group_id=g.id)
```

Comment:
Unit test fix, this has broken awhile back, but I have been bad about running the unit tests after making changes so I hadn't noticed :(